resolvers.ts

```diff
@@ -193,27 +194,143 @@ export const resolvers: Resolvers = {
       });
       return pathogens;
     },
-    Samples: async (root, args, context, info) => {
-      if (args.sampleId) {
-        const sample = await get(
-          `/samples/${args.sampleId}.json`,
-          args,
-          context
-        );
-        if (args.projectId && sample.project.id !== parseInt(args.projectId)) {
-          return notFound(
-            `Sample ${args.sampleId} not found in project ${args.projectId}`
-          );
-        }
-        return formatSample(sample);
-      } else if (args.projectId) {
-        const { samples } = await get(
-          `/samples/index_v2.json?projectId=${args.projectId}&snapshotShareId=&basic=true`,
-          args,
-          context
-        );
-        return formatSamples(samples);
+    samples: async (root, args, context) => {
```

Comment:
Any reason you went with lower case rather than upper case? Closer to next gen functionality?

Reply:
Exactly, otherwise when we start exposing NextGen directly we'll have to change the cases